styleguide/source/_patterns/01-atoms/button.json

```diff
@@ -0,0 +1,9 @@
+{
+  "button": {
+    "href": "",
+    "info": "alt",
+    "text": "Button",
+    "style": "solid",
+    "size": "normal"

```

Comment:
This is missing the button type definition. Also "normal" seems redundant and unnecessary. If these are the defaults then those should just be left blank so that we don't add unnecessary classes later on.